### PR TITLE
Fixes #1631 Pint parameters

### LINKED ISSUE #1631: Set Pint calibration parameters to visible
s. https://github.com/Open-Systems-Pharmacology/Forum/issues/668#issuecomment-761093547

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: src/Db/Diff/PKSimDB_diff.txt [hunk 1]
```diff
@@ -1 +1,3 @@
-UPDATE tab_container_names SET icon_name='Endosome' WHERE container_type='COMPARTMENT' AND container_name='Endosome';
+UPDATE tab_calculation_method_parameter_rates SET group_name='PERMEABILITY_CALIBRATION', read_only=0, visible=1 WHERE parameter_id=228;
+UPDATE tab_calculation_method_parameter_rates SET group_name='PERMEABILITY_CALIBRATION', read_only=0, visible=1 WHERE parameter_id=230;
```
Comment (msevestre) [inline src/Db/Diff/PKSimDB_diff.txt:2]: Parameters are visible and not readonly anymore

File: src/Db/Diff/PKSimDB_diff.txt [hunk 2]
```diff
@@ -1 +1,3 @@
-UPDATE tab_container_names SET icon_name='Endosome' WHERE container_type='COMPARTMENT' AND container_name='Endosome';
+UPDATE tab_calculation_method_parameter_rates SET group_name='PERMEABILITY_CALIBRATION', read_only=0, visible=1 WHERE parameter_id=228;
+UPDATE tab_calculation_method_parameter_rates SET group_name='PERMEABILITY_CALIBRATION', read_only=0, visible=1 WHERE parameter_id=230;
+INSERT INTO tab_groups(group_name,display_name,description,parent_group,icon_name,visible,sequence,is_advanced,unique_id,pop_display_name,full_name) VALUES('PERMEABILITY_CALIBRATION','Intestinal permeability calibration','Intestinal permeability calibration','PERMEABILITY',NULL,1,6,1,152,NULL,'Intestinal permeability calibration');
```
Comment (msevestre) [inline src/Db/Diff/PKSimDB_diff.txt:3]: A new group. `Intestinal permeability calibration". Let's discuss quickly if we are happy with the name or not

Comment (Yuri05) [reply src/Db/Diff/PKSimDB_diff.txt:3]: I just created a (empty) Wiki page, where we can add the detailed description how to use this calibration:
https://github.com/Open-Systems-Pharmacology/PK-Sim/wiki/Intestinal-Permeability-calibration
Can you add this link to the Description of the new group (will be  shown as a tooltip then)?

Comment (Yuri05) [reply src/Db/Diff/PKSimDB_diff.txt:3]: Regarding the name: I personally is happy with it, but maybe @AndreDlm can comment on this

Comment (msevestre) [reply src/Db/Diff/PKSimDB_diff.txt]: done. Andrea likes the name also

File: src/Db/Diff/PKSimDB_diff.txt [hunk 3]
```diff
@@ -1 +1,5 @@
...
+INSERT INTO tab_groups(group_name,display_name,description,parent_group,icon_name,visible,sequence,is_advanced,unique_id,pop_display_name,full_name) VALUES('PERMEABILITY_CALIBRATION','Intestinal permeability calibration','Parameters used to correlate experimental with in-silico intestinal permeability. Please refer to https://github.com/Open-Systems-Pharmacology/PK-Sim/wiki/Intestinal-Permeability-calibration for details.','PERMEABILITY',NULL,1,6,1,152,NULL,'Intestinal permeability calibration');
+UPDATE tab_parameters SET description='Parameter b for correlation of intestinal permeability (transcellular). Please refer to https://github.com/Open-Systems-Pharmacology/PK-Sim/wiki/Intestinal-Permeability-calibration for details.' WHERE parameter_name='Parameter b for correlation of intestinal permeability (transcellular)';
+UPDATE tab_parameters SET description='Parameter m for correlation of intestinal permeability (transcellular). Please refer to https://github.com/Open-Systems-Pharmacology/PK-Sim/wiki/Intestinal-Permeability-calibration for details.' WHERE parameter_name='Parameter m for correlation of intestinal permeability (transcellular)';
```
Comment (msevestre) [inline src/Db/Diff/PKSimDB_diff.txt]: @Yuri05 I have also updated the description here. Unfortunately, the tooltip is cut for group I am not sure why

File: src/Db/Dump/PKSimDB_dump.txt [hunk 4]
```diff
@@ -65545,6 +65545,7 @@ INSERT INTO tab_groups VALUES('VOLUME_INTRACELLULAR','Intracellular','Intracellu
 INSERT INTO tab_groups VALUES('VOLUME_MUCOSA','Mucosa (total)','Mucosa (total)','ORGAN_VOLUMES',NULL,1,1,1,129,'','Organ volumes - Mucosa (total)');
 INSERT INTO tab_groups VALUES('VOLUME_PLASMA','Plasma','Plasma','ORGAN_VOLUMES',NULL,1,1,1,126,'','Organ volumes - Plasma');
 INSERT INTO tab_groups VALUES('VOLUME_VASCULAR_ENDOTHELIUM','Vascular endothelium','Vascular endothelium','ORGAN_VOLUMES',NULL,1,1,1,127,'','Organ volumes - Vascular endothelium');
+INSERT INTO tab_groups VALUES('PERMEABILITY_CALIBRATION','Intestinal permeability calibration','Parameters used to correlate experimental with in-silico intestinal permeability. Please refer to permeability.open-systems-pharmacology.org for details.','PERMEABILITY',NULL,1,6,1,152,NULL,'Intestinal permeability calibration');
```
Comment (msevestre) [inline src/Db/Dump/PKSimDB_dump.txt:65548]: ok I created a shortcut...should be better

Comment (Yuri05) [reply src/Db/Dump/PKSimDB_dump.txt:65548]: nice!

**Review by Yuri05 — APPROVED** [review]

**Review by AndreDlm — COMMENTED** [review]
fine with me

**Pull request merged**